### ISSUE TITLE
Crash in call to accessibilityElements

### ISSUE THREAD

**Issue by alimoeeny** [TTTAttributedLabel/TTTAttributedLabel#577]
We are stilling seeing a few crashes in **TTTAttributedLabel.m line 1301** `-[TTTAttributedLabel accessibilityElements]`
I know it has been discussed before here and may have been mostly fixed, but we are using the latests Pod and get a few crashes each week. Any suggestions on this? 
On another note, is there any merit in releasing a new pod release?

**Comment by getaaron** [comment]
There haven't been any changes since 2.0.0 that would affect the `accessibilityElements` method. I would be happy to look at any reproduction steps, crash logs, etc. you have. If you can write a test to reproduce the failure that would be really helpful.

**Comment by alimoeeny** [comment]
Thanks @getaaron , I don't think I'd be able to reproduce it. I have a bunch of crash reports but they are pretty generic. 
I'll get back to you if I got more info. 
Cheers,

**Comment by getaaron** [comment]
:+1:

**Comment by glentregoning** [comment]
FYI I'm still seeing a small but solid number of crashes here. 

Fatal Exception: NSRangeException
-[__NSCFString substringWithRange:]: Range {9223372036854775807, 0} out of bounds; string length 23
-[TTTAttributedLabel accessibilityElements]
 raw

The issue is 100% on ios9. I've seen no crashes on ios8.

**Comment by Kuler29** [comment]
In my crash happens after following these steps:
1) In storyboard we have a TTTAttributedLabel with plain text
2) Add link to label
3) Set label's attributedText to something shorter than original text
4) Try to access accessibilityElements property or, actually, just running UI test at this time causes a crash.

In attachment, there is a simple project with the described issue

[AttributedLabelAccessibilityException.zip](https://github.com/TTTAttributedLabel/TTTAttributedLabel/files/55989/AttributedLabelAccessibilityException.zip)
